examples/mnist/mnist_deep_simplified.py

```diff
@@ -125,10 +126,22 @@ def bias_variable(shape):
 
 def train_mnist_cnn(FLAGS):
     # Config
-    config = tf.ConfigProto(
-        allow_soft_placement=True,
-        log_device_placement=False,
-        inter_op_parallelism_threads=1)
+    # Enable the custom optimizer using the rewriter config options
+    if (FLAGS.use_grappler):
+        rewrite_options = rewriter_config_pb2.RewriterConfig(custom_optimizers=[
+            rewriter_config_pb2.RewriterConfig.CustomGraphOptimizer(
+                name="ngraph-optimizer")
+        ])
+        config = tf.ConfigProto(
+            allow_soft_placement=True,
+            log_device_placement=False,
+            inter_op_parallelism_threads=1,
+            graph_options=tf.GraphOptions(rewrite_options=rewrite_options))
+    else:
+        config = tf.ConfigProto(
+            allow_soft_placement=True,
+            log_device_placement=False,
+            inter_op_parallelism_threads=1)
```

Comment:
We can minimize the amount of code change by doing this. This would be an example on the example minimal code change required to use grappler style ngtf: Just add a new field to some existing ConfigProto

```
config = tf.ConfigProto(
            allow_soft_placement=True,
            log_device_placement=False,
            inter_op_parallelism_threads=1)
if (FLAGS.use_grappler):
    rewrite_options = rewriter_config_pb2.RewriterConfig(custom_optimizers=[
            rewriter_config_pb2.RewriterConfig.CustomGraphOptimizer(
                name="ngraph-optimizer")
        ])
    config.MergeFrom(tf.ConfigProto(graph_options=tf.GraphOptions(rewrite_options=rewrite_options)))
```